lib/ansible/plugins/lookup/aws_ssm.py

```diff
@@ -45,130 +48,150 @@
   debug: msg="{{ lookup('aws_ssm', 'NoKey') }}"
 
 - name: lookup ssm parameter store in nominated region
-  debug: msg="{{ lookup('aws_ssm', 'Hello', 'region=us-east-2' ) }}"
+  debug: msg="{{ lookup('aws_ssm', 'Hello', region=us-east-2 ) }}"
 
 - name: lookup ssm parameter store without decrypted
-  debug: msg="{{ lookup('aws_ssm', 'Hello', 'decrypt=False' ) }}"
+  debug: msg="{{ lookup('aws_ssm', 'Hello', decrypt=False ) }}"
 
 - name: lookup ssm parameter store in nominated aws profile
-  debug: msg="{{ lookup('aws_ssm', 'Hello', 'aws_profile=myprofile' ) }}"
+  debug: msg="{{ lookup('aws_ssm', 'Hello', aws_profile=myprofile ) }}"
 
 - name: lookup ssm parameter store with all options.
-  debug: msg="{{ lookup('aws_ssm', 'Hello', 'decrypt=false', 'region=us-east-2', 'aws_profile=myprofile') }}"
+  debug: msg="{{ lookup('aws_ssm', 'Hello', decrypt=false, region=us-east-2, aws_profile=myprofile') }}"
 
 - name: return a dictionary of ssm parameters from a hierarchy path
-  debug: msg="{{ lookup('aws_ssm', '/PATH/to/params', 'region=ap-southeast-2', 'bypath', 'recursive=true' ) }}"
+  debug: msg="{{ lookup('aws_ssm', '/PATH/to/params', region=ap-southeast-2, bypath=true, recursive=true' ) }}"
 
 - name: return a dictionary of ssm parameters from a hierarchy path with shortened names (param instead of /PATH/to/param)
-  debug: msg="{{ lookup('aws_ssm', '/PATH/to/params', 'region=ap-southeast-2', 'shortnames', 'bypath', 'recursive=true' ) }}"
+  debug: msg="{{ lookup('aws_ssm', '/PATH/to/params', region=ap-southeast-2, shortnames, bypath=true, recursive=true ) }}"
 
 - name: Iterate over a parameter hierarchy
   debug: msg='key contains {{item.Name }} with value {{item.Value}} '
   with_aws_ssm:
-    - '/TEST/test-list'
-    - 'region=ap-southeast-2'
-    - 'bypath'
+    - '/TEST/test-list region=ap-southeast-2, bypath'
```

Comment:
This example will not work.  Use loop with lookup/query/q instead.  Any of the following will be fine:
```
loop: '{{ lookup("aws_ssm", "/TEST/test-list", region="ap-southeast-2", bypath=true, wantlist=true) }}'
loop: '{{ query("aws_ssm", "/TEST/test-list", region="ap-southeast-2", bypath=true) }}'
loop: '{{ q("aws_ssm", "/TEST/teset-list", region="ap-southeast-2", bypath=true) }}'
```